ros_cross_compile/dependencies.py

```diff
@@ -92,7 +92,7 @@ def assert_install_rosdep_script_exists(
     ros_workspace_dir: Path,
     platform: Platform
 ) -> bool:
-    install_rosdep_script_path = ros_workspace_dir / rosdep_install_script(platform)
+    install_rosdep_script_path = ros_workspace_dir / str(rosdep_install_script(platform))
```

Comment:
I don't think I am in favor of these changes. It forces mypy to be happier, but I'd like to figure out what changed that actually made it stop working, instead of changing already-correct code to appease a linter

At least - that is based on my assumption that `Path / Path` is valid code based on the language in --event-handlers console_cohesion+ console_package_list+

Reply:
Actually I opened https://github.com/ros-tooling/cross_compile/pull/326, we'll see - I think it doesn't fix 3.5. We can discuss there.

Reply:
Fair enough. Removed.